specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json

```diff
@@ -4944,6 +5195,20 @@
         "id"
       ]
     },
+    "SqlStoredProcedureResource": {
```

Comment:
Calling this a "resource" with an "id" property could make things confusing, as this isn't an Azure resource. Please could you update the naming to avoid confusion?